### PR TITLE
ci workflow env variables

### PR DESCRIPTION
**Pull request by ggabernet**
Adding env variables to the env scope in the pipeline template

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by ggabernet** [comment]
Thanks, I will just wait for @ewels approval, as last time we were too quick merging

**Comment by MaxUlysse** [comment]
Makes sense

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/ci.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ on: [push, pull_request]
 
 jobs:
   test:
+    env: 
+      NXF_VER: {% raw %}${{ matrix.nxf_ver }}{% endraw %}
```
Comment (ewels) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/ci.yml]: I think we can get rid of the `raw` stuff and the `$` symbol here..

Just `NXF_VER: {{ matrix.nxf_ver }}` should be fine I think.

Comment (ggabernet) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/ci.yml]: Thanks, corrected that now

**Comment by ewels** [comment]
oops, bit trigger happy on the merge there sorry.. Hopefully this was good to go though!

I guess I was wrong and we did need the `{% raw %}` stuff? I was thinking that it was for Actions, but looking now that I see that this is for the Jinja2 templating, which makes more sense (so it won't appear in the final Actions template..)

**Comment by ggabernet** [comment]
yes, all good! We did need the {% raw %} stuff because of the Jinja2 templating, exactly!

**Pull request merged**